src/Simulation/Simulation.jl

```diff
@@ -706,24 +706,6 @@ Takes the current state of `sim.electric_potential` and refines it with respect
 * `sim::Simulation{T}`: [`Simulation`](@ref) for which `sim.electric_potential` will be refined.
 * `max_diffs::Tuple{<:Real,<:Real,<:Real}`: Maximum potential difference between two discrete ticks of `sim.electric_potential.grid` after refinement.
 * `minimum_distances::Tuple{<:Real,<:Real,<:Real}`: Minimum distance (in SI Units) between two discrete ticks of `sim.electric_potential.grid` after refinement.
-
-## Keywords
```

Comment:
Why delete the `Keywords` here?
Do you refer to where the explanation for the keywords can be found?

Reply:
Because they are not used in those functions. 

There explanation still exists in the docstrings where they are exists.